src/data/articles/units/units.json

```diff
@@ -4,15 +4,157 @@
+            "{{ units.guardNaki.recSpeed }}",
+            "{{ units.guardNaki.speed }}"
+          ],
+          [
+            "{{ units.elderSpirit.name }}",
+            "{{ units.elderSpirit.attack }}",
+            "{{ units.elderSpirit.defense }}",
+            "{{ units.elderSpirit.recSpeed }}",
+            "{{ units.elderSpirit.speed }}"
+          ],
+          [
+            "{{ units.marshy.name }}",
+            "{{ units.marshy.attack }}",
+            "{{ units.marshy.defense }}",
+            "{{ units.marshy.recSpeed }}",
+            "{{ units.marshy.speed }}"
+          ],
+          [
+            "{{ units.pangoan.name }}",
+            "{{ units.pangoan.attack }}",
+            "{{ units.pangoan.defense }}",
+            "{{ units.pangoan.recSpeed }}",
+            "{{ units.pangoan.speed }}"
+          ],
+          [
+            "{{ units.wasty.name }}",
+            "{{ units.wasty.attack }}",
+            "{{ units.wasty.defense }}",
+            "{{ units.wasty.recSpeed }}",
+            "{{ units.wasty.speed }}"
+          ],
+          [
+            "{{ Athlas # article/units/athlas }}",
+            "{{ units.athlas.attack }}",
+            "{{ units.athlas.defense }}",
+            "{{ units.athlas.recSpeed }}",
+            "{{ units.athlas.speed }}"
+          ]
+        ]
+      }
     },
     {
       "subtitle": "Special Units",
-      "paragraph": "Raider Naki \n {{ Teryx # article/units/teryx }} \n Ovivi"
+      "table": {
+        "header": [
+          "Name",
+          "Attack",
+          "Defense",
+          "Recruitment Time",
+          "Speed (s / field)"
+        ],
+        "rows": [
+          [
+            "{{ units.raiderNaki.name }}",
+            "{{ units.raiderNaki.attack }}",
+            "{{ units.raiderNaki.defense }}",
+            "{{ units.raiderNaki.recSpeed }}",
+            "{{ units.raiderNaki.speed }}"
+          ],
+          [
+            "{{ Teryx # article/units/teryx }}",
+            "{{ units.teryx.attack }}",
+            "{{ units.teryx.defense }}",
+            "{{ units.teryx.recSpeed }}",
+            "{{ units.teryx.speed }}"
+          ],
+          [
+            "{{ units.ovivi.name }}",
+            "{{ units.ovivi.attack }}",
+            "{{ units.ovivi.defense }}",
+            "{{ units.ovivi.recSpeed }}",
+            "{{ units.ovivi.speed }}"
+          ]
+        ]
+      }
     },
     {
       "subtitle": "Hero Units",
-      "paragraph": "Raider Hero \n Teryx Hero \n Ovivi Hero"
+      "table": {
+        "header": [
+          "Name",
+          "Attack",
+          "Defense",
+          "Recruitment Time",
+          "Speed (s / field)"
+        ],
+        "rows": [
+          [
+            "{{ units.raiderHero.name }}",
+            "{{ units.raiderHero.attack }}",
+            "{{ units.raiderHero.defense }}",
+            "{{ units.raiderHero.recSpeed }}",
+            "{{ units.raiderHero.speed }}"
```

Comment:
So far i can't find this information visually displayed in the "units-overview" of the wiki. Is this intentionally not yet displayed and just does to show how dynamic tables are used?